app/lib/BaseEditorController.php

```diff
@@ -732,54 +768,56 @@ public function Summary($pa_options=null) {
 		$this->view->setVar('bundle_displays', $va_displays);
 		$this->view->setVar('t_display', $t_display);
 
-		// Check validity and access of specified display
-		if ($t_display->load($vn_display_id) && ($t_display->haveAccessToDisplay($this->request->getUserID(), __CA_BUNDLE_DISPLAY_READ_ACCESS__))) {
-			$this->view->setVar('display_id', $vn_display_id);
+		if (!$pb_ajax_load_displays || $this->request->isAjax()) {
```

Comment:
Can we invert this check and drop out of function instead? so it doesn't make a change on every single line